frontend/src/Components/NewBook.tsx

```diff
@@ -0,0 +1,32 @@
+import {ChangeEvent, useState} from "react";
+
+type NewBookProps = {
+    newBook : (title:string, author:string, isbn:string) => void
+}
+export default function NewBook(props : NewBookProps) {
+
+    const [title, setTitle] = useState("")
+    const [author, setAuthor] = useState("")
+    const [isbn, setIsbn] = useState("")
+
+    const onTitleChange = (event: ChangeEvent<HTMLInputElement>) => {
+        setTitle(event.target.value)
+    }
+ const onAuthorChange = (event: ChangeEvent<HTMLInputElement>) => {
+        setAuthor(event.target.value)
+    }
+ const onIsbnChange = (event: ChangeEvent<HTMLInputElement>) => {
+        setIsbn(event.target.value)
+    }
+
+    return (
+        <div>
+            <input onChange={onTitleChange} value={title} />
```

Comment:
Hier hättest du auch die Kurzschreibweise nutzen können und die arrowfunction direkt in die onChange schreiben können.
Ist aber wieder meckern auf hohem Niveau, das Auslagern ist eine valide Lösung und sehr übersichtlich :-)